nixos/modules/services/web-servers/tomcat.nix

```diff
@@ -185,6 +188,10 @@ in
       after = [ "network.target" ];
 
       preStart = ''
+        # Delete the existing base directory, if any
+        # (we don't want remainders of old configuration!)
+        rm -rf ${cfg.baseDir}
```

Comment:
Would this not also delete logs?

Reply:
Right, it would, and it's definitely not wanted.